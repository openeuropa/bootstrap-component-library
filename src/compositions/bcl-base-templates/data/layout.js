import headerSimple from "@openeuropa/bcl-data-header/data--simple";
import headerNoForm from "@openeuropa/bcl-data-header/data--no-form";
import footer from "@openeuropa/bcl-data-footer/data";
import header_ucpkn from "@openeuropa/bcl-data-header/data--ucpkn";
import header_joinup from "@openeuropa/bcl-data-header/data--neutral";

export default {
  headerSimple,
  headerNoForm,
  footer,
  header_ucpkn,
  header_joinup,
};
