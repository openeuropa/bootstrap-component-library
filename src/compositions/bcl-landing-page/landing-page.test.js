import { renderTwigFileAsNode } from "@openeuropa/bcl-test-utils";

import layout from "@openeuropa/bcl-base-templates/data/layout";
import demoData from "@openeuropa/bcl-landing-page/data";

const template = "@oe-bcl/bcl-landing-page/landing-page.html.twig";
const render = (params) => renderTwigFileAsNode(template, params);

const dataLanding = {
  content_type: "landing-page",
  header: layout.headerSimple,
  footer: layout.footer,
  with_banner: true,
  with_header: true,
  with_footer: true,
  ...demoData,
};

describe("OE - Landing page", () => {
  test("renders correctly", () => {
    expect.assertions(1);

    return expect(render(dataLanding)).resolves.toMatchSnapshot();
  });
});
