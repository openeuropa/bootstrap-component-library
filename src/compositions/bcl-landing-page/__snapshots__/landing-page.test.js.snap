// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OE - Landing page renders correctly 1`] = `
<jest>
  <header
    class="bcl-header bcl-header--eu"
  >
    <!-- navbar_settings -->
    <nav
      aria-label="Top Navigation"
      class="w-100 shadow-sm navbar navbar-expand-lg navbar-light"
    >
      <div
        class="container"
      >
        <a
          class="navbar-brand"
          href=""
        >
          <img
            alt="desktop european union logo"
            class="d-none d-lg-block"
            src="https://cdn1.fpfis.tech.ec.europa.eu/ecl/v3.0.2/eu/images/logo/standard-version/positive/logo-eu--en.svg"
          />
          <img
            alt="mobile european union logo"
            class="d-lg-none"
            src="https://cdn1.fpfis.tech.ec.europa.eu/ecl/v3.0.2/eu/images/logo/condensed-version/positive/logo-eu--en.svg"
          />
        </a>
        <ul
          class="nav"
        >
          <li
            class="nav-item"
          >
            <a
              class="text-center nav-link"
              data-bs-target="#languageModal"
              data-bs-toggle="modal"
              href="#"
            >
              <svg
                class="me-lg-2-5 bi icon--xs"
              >
                <use
                  xlink:href="/icons.svg#chat-left-dots-fill"
                />
              </svg>
              <span
                class="d-block d-lg-inline-block"
              >
                English
              </span>
            </a>
          </li>
          <li
            class="nav-item"
          >
            <a
              class="text-center nav-link"
              data-bs-target="#loginModal"
              data-bs-toggle="modal"
              href="#"
            >
              <svg
                class="me-lg-2-5 bi icon--xs"
              >
                <use
                  xlink:href="/icons.svg#person-fill"
                />
              </svg>
              <span
                class="d-block d-lg-inline-block"
              >
                Log in
              </span>
            </a>
          </li>
        </ul>
      </div>
    </nav>
    <!-- project name -->
    <!-- navbar -->
    <nav
      aria-label="main-navigation"
      class="bcl-header__navbar navbar navbar-expand-lg navbar-dark"
    >
      <div
        class="container"
      >
        <button
          aria-controls="navbarNavDropdown"
          aria-expanded="false"
          aria-label="Toggle navigation"
          class="navbar-toggler"
          data-bs-target="#navbarNavDropdown"
          data-bs-toggle="collapse"
          type="button"
        >
          <span
            class="navbar-toggler-icon"
          />
        </button>
        <div
          class="collapse navbar-collapse"
          id="navbarNavDropdown"
        >
          <ul
            class="me-auto navbar-nav"
          >
            <li
              class="nav-item"
            >
              <a
                aria-current="page"
                aria-label="homepage"
                class="nav-link active"
                href="/example.html"
              >
                Home
              </a>
            </li>
            <li
              class="nav-item"
            >
              <a
                class="nav-link"
                href="/example.html"
              >
                Item 01
              </a>
            </li>
            <li
              class="nav-item"
            >
              <a
                class="nav-link"
                href="/example.html"
              >
                Item 02
              </a>
            </li>
            <li
              class="nav-item dropdown"
            >
              <a
                aria-expanded="false"
                class="nav-link dropdown-toggle"
                data-bs-toggle="dropdown"
                href="#"
                id="navbarDropdownMenuLink"
                role="button"
              >
                Dropdown link
              </a>
              <ul
                aria-labelledby="navbarDropdownMenuLink"
                class="dropdown-menu"
              >
                <li>
                  <a
                    class="dropdown-item"
                    href="/example.html"
                  >
                    Action
                  </a>
                </li>
                <li>
                  <a
                    class="dropdown-item"
                    href="/example.html"
                  >
                    Another action
                  </a>
                </li>
                <li>
                  <a
                    class="dropdown-item"
                    href="/example.html"
                  >
                    Something else here
                  </a>
                </li>
              </ul>
            </li>
          </ul>
          <form
            class="d-flex mt-3 mt-lg-0"
          >
            <label
              class="form-label visually-hidden"
              for="inlineFormInputGroupSearch"
            >
              Search
            </label>
            <input
              class="border-start-0 rounded-0 rounded-start form-control"
              id="inlineFormInputGroupSearch"
              placeholder="Search"
              required="true"
              type="text"
            />
            <button
              class="border-start-0 rounded-0 rounded-end btn btn-light btn-md"
              type="button"
            >
              <span
                class="visually-hidden"
              >
                search
              </span>
              <svg
                class="bi icon--fluid"
              >
                <use
                  xlink:href="/icons.svg#search"
                />
              </svg>
            </button>
          </form>
        </div>
      </div>
    </nav>
    <!-- breadcrumbs -->
    <div
      class="container"
    >
      <nav
        aria-label="breadcrumb"
        class="mt-3"
      >
        <ol
          class="breadcrumb"
        >
          <li
            class="breadcrumb-item d-none d-md-block"
          >
            <a
              aria-label="homepage"
              href="/example.html"
            >
              Home
            </a>
          </li>
          <li
            class="breadcrumb-item d-none d-md-block"
          >
            <a
              href="/example.html"
            >
              European Union
            </a>
          </li>
          <li
            class="breadcrumb-item"
          >
            <a
              href="/example.html"
            >
              <svg
                class="d-md-none ms-0 me-2 me-2-5 bi icon--s"
              >
                <use
                  xlink:href="/icons.svg#arrow-left"
                />
              </svg>
              About the European Union
            </a>
          </li>
          <li
            aria-current="page"
            class="breadcrumb-item d-none d-md-block active"
          >
            News
          </li>
        </ol>
      </nav>
    </div>
  </header>
  <div
    aria-hidden="true"
    aria-label="loginModalLabel"
    class="modal fade"
    id="loginModal"
    tabindex="-1"
  >
    <div
      class="modal-dialog"
    >
      <div
        class="modal-content"
      >
        <div
          class="modal-header"
        >
          <h5
            class="modal-title"
            id="loginModalLabel"
          >
            Log in
          </h5>
          <button
            aria-label="Close"
            class="btn-close"
            data-bs-dismiss="modal"
            type="button"
          />
        </div>
        <div
          class="modal-body"
        >
          <h5>
            Disclaimer 1
          </h5>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum accumsan eget orci eu aliquet. Suspendisse potenti. Integer volutpat, nisl et placerat tincidunt
          </p>
          <h5>
            Disclaimer 2
          </h5>
          <p>
            Duis bibendum tristique lobortis. Nam luctus nisi sit amet turpis faucibus, ut dapibus elit sollicitudin. Phasellus a massa magna. Nullam ac imperdiet leo.
          </p>
        </div>
        <div
          class="modal-footer"
        >
          <button
            class="btn btn-secondary"
            data-bs-dismiss="modal"
            type="button"
          >
            Close
          </button>
          <button
            class="btn btn-primary"
            type="button"
          >
            Access EU login
          </button>
        </div>
      </div>
    </div>
  </div>
  <div
    aria-hidden="true"
    aria-label="languageModalLabel"
    class="bcl-language-list-modal bcl-language-list-modal--neutral modal fade"
    id="languageModal"
    tabindex="-1"
  >
    <div
      class="modal-dialog modal-xl modal-fullscreen-md-down"
    >
      <div
        class="modal-content"
      >
        <div
          class="modal-header"
        >
          <div
            class="d-flex align-items-center"
          >
            <svg
              class="bi icon--s"
            >
              <use
                xlink:href="/icons.svg#globe2"
              />
            </svg>
            <h5
              class="mb-0 ms-2"
            >
              Select your language
            </h5>
          </div>
          <button
            aria-label="Close"
            class="btn-close"
            data-bs-dismiss="modal"
            type="button"
          />
        </div>
        <div
          class="modal-body"
        >
          <div
            class="bcl-language-list bcl-language-list--neutral"
          >
            <div
              class="main pt-4"
            >
              <div
                class="pb-2"
              >
                <div
                  class="row"
                >
                  <div
                    class="col-12 col-md"
                  >
                    <a
                      class="w-100 mb-2-5 btn justify-content-start"
                      href="#"
                      hreflang="bg"
                      lang="bg"
                    >
                      български
                    </a>
                    <a
                      class="w-100 mb-2-5 btn justify-content-start"
                      href="#"
                      hreflang="es"
                      lang="es"
                    >
                      español
                    </a>
                    <a
                      class="w-100 mb-2-5 btn justify-content-start"
                      href="#"
                      hreflang="cs"
                      lang="cs"
                    >
                      čeština
                    </a>
                    <a
                      class="w-100 mb-2-5 btn justify-content-start"
                      href="#"
                      hreflang="da"
                      lang="da"
                    >
                      dansk
                    </a>
                    <a
                      class="w-100 mb-2-5 btn justify-content-start"
                      href="#"
                      hreflang="de"
                      lang="de"
                    >
                      Deutsch
                    </a>
                    <a
                      class="w-100 mb-2-5 btn justify-content-start"
                      href="#"
                      hreflang="et"
                      lang="et"
                    >
                      eesti
                    </a>
                    <a
                      class="w-100 mb-2-5 btn justify-content-start"
                      href="#"
                      hreflang="el"
                      lang="el"
                    >
                      ελληνικά
                    </a>
                    <a
                      class="w-100 mb-2-5 btn selected justify-content-between"
                      href="#"
                      hreflang="en"
                      lang="en"
                    >
                      English
                      <svg
                        class="ms-2-5 bi icon--s"
                      >
                        <use
                          xlink:href="/icons.svg#check2"
                        />
                      </svg>
                    </a>
                    <a
                      class="w-100 mb-2-5 btn justify-content-start"
                      href="#"
                      hreflang="fr"
                      lang="fr"
                    >
                      français
                    </a>
                    <a
                      class="w-100 mb-2-5 btn justify-content-start"
                      href="#"
                      hreflang="ga"
                      lang="ga"
                    >
                      Gaeilge
                    </a>
                    <a
                      class="w-100 mb-2-5 btn justify-content-start"
                      href="#"
                      hreflang="hr"
                      lang="hr"
                    >
                      hrvatski
                    </a>
                    <a
                      class="w-100 mb-2-5 btn justify-content-start"
                      href="#"
                      hreflang="it"
                      lang="it"
                    >
                      italiano
                    </a>
                  </div>
                  <div
                    class="col-12 col-md"
                  >
                    <a
                      class="w-100 mb-2-5 btn justify-content-start"
                      href="#"
                      hreflang="lv"
                      lang="lv"
                    >
                      latviešu
                    </a>
                    <a
                      class="w-100 mb-2-5 btn justify-content-start"
                      href="#"
                      hreflang="lt"
                      lang="lt"
                    >
                      lietuvių
                    </a>
                    <a
                      class="w-100 mb-2-5 btn justify-content-start"
                      href="#"
                      hreflang="hu"
                      lang="hu"
                    >
                      magyar
                    </a>
                    <a
                      class="w-100 mb-2-5 btn justify-content-start"
                      href="#"
                      hreflang="mt"
                      lang="mt"
                    >
                      Malti
                    </a>
                    <a
                      class="w-100 mb-2-5 btn justify-content-start"
                      href="#"
                      hreflang="nl"
                      lang="nl"
                    >
                      Nederlands
                    </a>
                    <a
                      class="w-100 mb-2-5 btn justify-content-start"
                      href="#"
                      hreflang="pl"
                      lang="pl"
                    >
                      polski
                    </a>
                    <a
                      class="w-100 mb-2-5 btn justify-content-start"
                      href="#"
                      hreflang="pt"
                      lang="pt"
                    >
                      português
                    </a>
                    <a
                      class="w-100 mb-2-5 btn justify-content-start"
                      href="#"
                      hreflang="ro"
                      lang="ro"
                    >
                      română
                    </a>
                    <a
                      class="w-100 mb-2-5 btn justify-content-start"
                      href="#"
                      hreflang="sk"
                      lang="sk"
                    >
                      slovenčina
                    </a>
                    <a
                      class="w-100 mb-2-5 btn justify-content-start"
                      href="#"
                      hreflang="sl"
                      lang="sl"
                    >
                      slovenščina
                    </a>
                    <a
                      class="w-100 mb-2-5 btn justify-content-start"
                      href="#"
                      hreflang="fi"
                      lang="fi"
                    >
                      suomi
                    </a>
                    <a
                      class="w-100 mb-2-5 btn justify-content-start"
                      href="#"
                      hreflang="sv"
                      lang="sv"
                    >
                      svenska
                    </a>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <main>
    <div
      class="bg-light text-dark text-center bcl-banner  overlay full-width"
    >
      <div
        class="bcl-banner__image"
        style="background-image:url(https://picsum.photos/id/154/1200/800)"
      />
      <div
        class="container"
      >
        <div
          class="bcl-banner__content border-secondary"
        >
          <div
            class="display-6 bcl-heading"
          >
            Demo banner title
          </div>
        </div>
      </div>
    </div>
    <div
      class="container mt-md-4-75 mt-4"
    >
      <div
        class="row"
      >
        <div
          class="col-12"
        >
          <div
            class="container"
          >
            <div
              class="my-4"
            >
              <h2
                class="mb-4"
              >
                Lorem ipsum dolor sit amet.
              </h2>
              <div
                class="row"
              >
                <div
                  class="col-12 col-md-6"
                >
                  <p>
                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
                  </p>
                  <p>
                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
                  </p>
                  <a
                    class="d-block mb-4"
                    href="#"
                  >
                    Read more
                  </a>
                </div>
                <div
                  class="col-12 col-md-6"
                >
                  <figure
                    class="rounded overflow-hidden"
                    src="https://www.youtube.com/embed/VFYlYeFAEH4"
                  >
                    <div
                      class="ratio ratio-16x9"
                    >
                      <iframe
                        allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
                        allowfullscreen=""
                        frameborder="0"
                        height="197"
                        src="https://www.youtube.com/embed/fgi-GSCB6ho"
                        title="New digital strategy first"
                        width="350"
                      />
                    </div>
                    <figcaption
                      class="bg-light p-3"
                    >
                      Media description text goes here.
                    </figcaption>
                  </figure>
                </div>
              </div>
            </div>
            <div
              class="my-4"
            >
              <div
                class="bcl-listing bcl-listing--highlight-3-col"
              >
                <h2
                  class="mb-4 bcl-heading"
                >
                  Listing item block title
                </h2>
                <div
                  class="row row-cols-1 row-cols-md-3 g-4"
                >
                  <div
                    class="col"
                  >
                    <article
                      class="listing-item--highlight border-0 bg-lighter h-100 rounded-2 card"
                    >
                      <img
                        alt="alt img"
                        class="card-img-top mb-3"
                        src="https://picsum.photos/seed/1002/600/400"
                      />
                      <div
                        class="card-body pt-0"
                      >
                        <h1
                          class="fs-5 card-title bcl-heading"
                        >
                          <a
                            class="standalone"
                            href="/example"
                          >
                            Listing item title
                          </a>
                        </h1>
                        <p
                          class="card-text mb-2"
                        >
                          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus ut ex tristique.
                        </p>
                      </div>
                    </article>
                  </div>
                  <div
                    class="col"
                  >
                    <article
                      class="listing-item--highlight border-0 bg-lighter h-100 rounded-2 card"
                    >
                      <img
                        alt="alt img"
                        class="card-img-top mb-3"
                        src="https://picsum.photos/seed/1003/600/400"
                      />
                      <div
                        class="card-body pt-0"
                      >
                        <h1
                          class="fs-5 card-title bcl-heading"
                        >
                          <a
                            class="standalone"
                            href="/example"
                          >
                            Listing item title
                          </a>
                        </h1>
                        <p
                          class="card-text mb-2"
                        >
                          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus ut ex tristique.
                        </p>
                      </div>
                    </article>
                  </div>
                  <div
                    class="col"
                  >
                    <article
                      class="listing-item--highlight border-0 bg-lighter h-100 rounded-2 card"
                    >
                      <img
                        alt="alt img"
                        class="card-img-top mb-3"
                        src="https://picsum.photos/seed/1004/600/400"
                      />
                      <div
                        class="card-body pt-0"
                      >
                        <h1
                          class="fs-5 card-title bcl-heading"
                        >
                          <a
                            class="standalone"
                            href="/example"
                          >
                            Listing item title
                          </a>
                        </h1>
                        <p
                          class="card-text mb-2"
                        >
                          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus ut ex tristique.
                        </p>
                      </div>
                    </article>
                  </div>
                </div>
                <a
                  class="mt-3-5 d-inline-block"
                  href="/example"
                >
                  Listing item block CTA
                  <svg
                    class="ms-2-5 bi icon--xs"
                  >
                    <use
                      xlink:href="/icons.svg#chevron-right"
                    />
                  </svg>
                </a>
              </div>
            </div>
            <div
              class="my-4"
            >
              <div
                class="bg-light text-center mb-4-5 bcl-banner shade "
              >
                <div
                  class="bcl-banner__image"
                  style="background-image:url(https://picsum.photos/id/154/1200/800)"
                />
                <div
                  class="container"
                >
                  <div
                    class="bcl-banner__content"
                  >
                    <div
                      class="display-6 mb-3 bcl-heading"
                    >
                      EU Budget for the future
                    </div>
                    <p
                      class="mb-4"
                    >
                      Innovation, economy, environment and geopolitics
                    </p>
                    <a
                      class="btn btn-primary"
                      href="/example.html"
                    >
                      Subscribe
                      <svg
                        class="ms-2-5 bi icon--fluid"
                      >
                        <use
                          xlink:href="/icons.svg#chevron-right"
                        />
                      </svg>
                    </a>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="my-4"
            >
              <h2
                class="mb-4"
              >
                Activities
              </h2>
              <div
                class="row"
              >
                <div
                  class="col-12 col-md-6"
                >
                  <p>
                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
                  </p>
                  <p>
                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
                  </p>
                  <a
                    class="d-block mb-4"
                    href="#"
                  >
                    Read more
                  </a>
                </div>
                <div
                  class="col-12 col-md-6"
                >
                  <figure
                    class="rounded overflow-hidden"
                    src="https://www.youtube.com/embed/VFYlYeFAEH4"
                  >
                    <div
                      class="ratio ratio-16x9"
                    >
                      <iframe
                        allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
                        allowfullscreen=""
                        frameborder="0"
                        height="197"
                        src="https://www.youtube.com/embed/fgi-GSCB6ho"
                        title="New digital strategy"
                        width="350"
                      />
                    </div>
                    <figcaption
                      class="bg-light p-3"
                    >
                      Media description text goes here.
                    </figcaption>
                  </figure>
                </div>
              </div>
            </div>
            <div
              class="my-4"
            >
              <div
                class="bcl-listing bcl-listing--highlight-3-col"
              >
                <h2
                  class="mb-4 bcl-heading"
                >
                  What are those activities
                </h2>
                <div
                  class="row row-cols-1 row-cols-md-3 g-4"
                >
                  <div
                    class="col"
                  >
                    <article
                      class="listing-item--highlight border-0 bg-lighter h-100 rounded-2 card"
                    >
                      <img
                        alt="alt img"
                        class="card-img-top mb-3"
                        src="https://picsum.photos/seed/1005/600/400"
                      />
                      <div
                        class="card-body pt-0"
                      >
                        <h1
                          class="fs-5 card-title bcl-heading"
                        >
                          <a
                            class="standalone"
                            href="/example"
                          >
                            Capacity development
                          </a>
                        </h1>
                        <p
                          class="card-text mb-2"
                        />
                        <ul>
                          <li>
                            UCPM Training Programme
                          </li>
                          <li>
                            Exercises
                          </li>
                          <li>
                            Exchange of experts
                          </li>
                          <li>
                            UCPM Prevention and preparedness projects
                          </li>
                        </ul>
                        <p />
                      </div>
                    </article>
                  </div>
                  <div
                    class="col"
                  >
                    <article
                      class="listing-item--highlight border-0 bg-lighter h-100 rounded-2 card"
                    >
                      <img
                        alt="alt img"
                        class="card-img-top mb-3"
                        src="https://picsum.photos/seed/1006/600/400"
                      />
                      <div
                        class="card-body pt-0"
                      >
                        <h1
                          class="fs-5 card-title bcl-heading"
                        >
                          <a
                            class="standalone"
                            href="/example"
                          >
                            Science, Research & innovation
                          </a>
                        </h1>
                        <p
                          class="card-text mb-2"
                        >
                          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus ut ex tristique. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus ut ex tristique.
                        </p>
                      </div>
                    </article>
                  </div>
                  <div
                    class="col"
                  >
                    <article
                      class="listing-item--highlight border-0 bg-lighter h-100 rounded-2 card"
                    >
                      <img
                        alt="alt img"
                        class="card-img-top mb-3"
                        src="https://picsum.photos/seed/1007/600/400"
                      />
                      <div
                        class="card-body pt-0"
                      >
                        <h1
                          class="fs-5 card-title bcl-heading"
                        >
                          <a
                            class="standalone"
                            href="/example"
                          >
                            KN Horizontal activities
                          </a>
                        </h1>
                        <p
                          class="card-text mb-2"
                        >
                          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus ut ex tristique.
                        </p>
                      </div>
                    </article>
                  </div>
                </div>
                <a
                  class="mt-3-5 d-inline-block"
                  href="/example"
                >
                  Listing item block CTA
                  <svg
                    class="ms-2-5 bi icon--xs"
                  >
                    <use
                      xlink:href="/icons.svg#chevron-right"
                    />
                  </svg>
                </a>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </main>
  <footer
    class="mt-md-4-75 mt-4 bcl-footer bcl-footer--neutral"
  >
    <div
      class="container"
    >
      <div
        class="pt-4 pt-lg-5 row"
      >
        <div
          class="col-12 col-lg-4"
        >
          <p
            class="fw-bold mb-2"
          >
            [Project name]
          </p>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque in sagittis dolor.
          </p>
        </div>
        <div
          class="col-12 col-lg-4 pb-4 pb-lg-0"
        >
          <p
            class="fw-bold border-bottom pb-2 mb-2"
          >
            Contact [project name]
          </p>
          <p>
            Contact information of the DG
          </p>
          <p
            class="fw-bold border-bottom pb-2 mt-3 mb-2"
          >
            Follow us on social media
          </p>
          <div
            class="mb-1"
          >
            <a
              class="standalone"
              href="#"
            >
              <svg
                class="me-2-5 bi icon--xs"
              >
                <use
                  xlink:href="/icons.svg#facebook"
                />
              </svg>
              Facebook
            </a>
          </div>
          <div
            class="mb-1"
          >
            <a
              class="standalone"
              href="#"
            >
              <svg
                class="me-2-5 bi icon--xs"
              >
                <use
                  xlink:href="/icons.svg#twitter"
                />
              </svg>
              Twitter
            </a>
          </div>
          <div
            class="mb-1"
          >
            <a
              class="standalone"
              href="#"
            >
              <svg
                class="me-2-5 bi icon--xs"
              >
                <use
                  xlink:href="/icons.svg#linkedin"
                />
              </svg>
              Linkedin
            </a>
          </div>
        </div>
        <div
          class="col-12 col-lg-4"
        >
          <p
            class="fw-bold border-bottom pb-2 mb-2"
          >
            About us
          </p>
          <p>
            Information about the DG
          </p>
          <p
            class="fw-bold border-bottom pb-2 mb-2"
          >
            Optional links
          </p>
          <div
            class="mb-1"
          >
            <a
              class="standalone"
              href="#"
            >
              Link
            </a>
          </div>
          <div
            class="mb-1"
          >
            <a
              class="standalone"
              href="#"
            >
              Link
            </a>
          </div>
          <div
            class="mb-1"
          >
            <a
              class="standalone"
              href="#"
            >
              Link
            </a>
          </div>
          <div
            class="mb-1"
          >
            <a
              class="standalone"
              href="#"
            >
              Link
            </a>
          </div>
        </div>
      </div>
      <div
        class="pb-4 pb-lg-5 mt-4 mt-lg-5 bcl-footer__bordered-row row"
      >
        <div
          class="col-12 col-lg-4"
        >
          <p
            class="fw-bold pb-2 mb-2"
          >
            More information on:
          </p>
          <div
            class="mb-1"
          >
            <a
              class="standalone"
              href="#"
            >
              Link
            </a>
          </div>
          <div
            class="mb-1"
          >
            <a
              class="standalone"
              href="#"
            >
              Link
            </a>
          </div>
          <div
            class="mb-1"
          >
            <a
              class="standalone"
              href="#"
            >
              Link
            </a>
          </div>
          <div
            class="mb-1"
          >
            <a
              class="standalone"
              href="#"
            >
              Link
            </a>
          </div>
        </div>
      </div>
    </div>
  </footer>
</jest>
`;
